.github/workflows/test-on-droplets-matrix.yml

```diff
@@ -238,6 +238,13 @@ jobs:
             -d '{"persistent_vms": [], "instances": ["${{ matrix.check_vm.item_hash }}"]}' \
             "http://${DROPLET_IPV4}:4020/control/allocations"
 
+      - name: Fetch system usage endpoint
+        run: |
+          export DROPLET_IPV4="$(doctl compute droplet get aleph-vm-ci-${{ matrix.os_config.alias }}-${{ matrix.check_vm.alias }} --output json | ./.github/scripts/extract_droplet_ipv4.py)"
+          curl -X GET -H "Content-Type: application/json" \
+            "http://${DROPLET_IPV4}:4020/about/usage/system"
+
+
```

Comment:
I think that adding that check on the CI without check the results, doesn't provide so much inside about if it's working well or not. I think that having a unit test for it with some results covered is enough to check if it's working well.

Reply:
It check  that the endpoint doesn't crash in "real" conditions, which for me is important for the integration part of test suite and is complementary with the unit test, especially if we are going to mock the lib in the unit test.
I agree ideally  we should check the content here also but at the moment we can already check visually when there is a problem the whole droplet things need more automation

Reply:
This endpoint seems to not be so relevant, so just check that returns "something" and don't crash isn't so useful from my point of view, and also it adds more usage time on the Digital Ocean droplets. I think that we need to agree with the best way to execute the CI tests and with kind of tests.

Reply:
I had send a reply by e-mail but it seems it didn't pass. 

> I think that we need to agree with the best way to execute the CI tests and with kind of tests.

I'm guessing you meant 'which' and not 'with' ? For me unit tests and integration tests are complementary.
I understand we want to reign in on the DO cost but we are going to do more comprehensive testing on them in the future so we should look at other way  to reduce the cost. But if you feel that strongly about it I can remove it from this PR